lib/scss/components/stories/_stories.scss

```diff
@@ -114,9 +114,12 @@
   }
 
   .su-stories-paragraph__cta-region {
+    @include modular-spacing('padding-bottom', 5);
+    @include modular-spacing('margin-top', 3);
```

Comment:
We try to avoid margin-top where possible. Is there another approach that would work?

Reply:
Whoops. Margin-top is a hard habit to break, it turns out. lol. How about this?